Перенос файла с заданием: working_with_dropdown_list (#25)
zinka22/Selenium-cource: tasks/working_with_dropdown_list.md

## Задание к [working_with_dropdown_list](../solutions/working_with_dropdown_list.py)

### Работа с выпадающим списком

> Для этой задачи мы придумали еще один вариант капчи для роботов. Придется немного переобучить нашего робота,

Reviewer: ```suggestion
Для этой задачи мы придумали ещё один вариант капчи для роботов. Придётся немного переобучить нашего робота,
```

чтобы он справился с новым заданием.

Напишите код, который реализует следующий сценарий:

1) открыть страницу https://suninjuly.github.io/selects1.html;
2) посчитать сумму заданных чисел;

> 3) выбрать в выпадающем списке значение равное расчитанной сумме;

Reviewer: ```suggestion
3) выбрать в выпадающем списке равное рассчитанной сумме значение;
```

4) нажать кнопку "Submit";

> 5) если все сделано правильно и достаточно быстро (в этой задаче тоже есть ограничение по времени),

Reviewer: ```suggestion
5) если всё сделано правильно и достаточно быстро (в этой задаче тоже есть ограничение по времени),
```

   вы увидите окно с числом. Отправьте полученное число в качестве ответа для этого задания.

Когда ваш код заработает, попробуйте запустить его на странице https://suninjuly.github.io/selects2.html.

> Ваш код и для нее тоже должен пройти успешно.

Reviewer: ```suggestion
Ваш код и для неё тоже должен пройти успешно.
```

> Подсказка:если вы получаете ошибку в духе "argument of type 'int' is not iterable", перепроверьте тип переменной,
> которую вы передаете в функцию поиска. Нужно передать строку.

Reviewer: ```suggestion
Подсказка: если вы получаете ошибку в духе "argument of type 'int' is not iterable", перепроверьте тип переменной,
которую вы передаёте в функцию поиска — нужно передавать строку.
```
